exchangelib/properties.py

```diff
@@ -326,7 +326,7 @@ class Mailbox(EWSElement):
     FIELDS = [
         TextField('name', field_uri='Name'),
         EmailAddressField('email_address', field_uri='EmailAddress'),
-        ChoiceField('routing_type', field_uri='RoutingType', choices={Choice('SMTP')}, default='SMTP'),
+        ChoiceField('routing_type', field_uri='RoutingType', choices={Choice('SMTP'), Choice('EX')}, default='SMTP'),
```

Comment:
(different bug) ```ValueError: Invalid choice 'EX' for field 'routing_type'. Valid choices are: SMTP```
and while yes, [Exchange documentation](https://docs.microsoft.com/en-us/exchange/client-developer/web-service-reference/mailbox) (which is terrible) says the only valid value for this field is SMTP, other [Exchange documentation](https://docs.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-perform-grouped-searches-by-using-ews-in-exchange) has examples of `EX` as a valid value here, so...?

Reply:
That is frustrating! Exchange needs to get its act together! Good researching on your part!